lib/readGltf.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+var path = require('path');
+var fs = require('fs');
+var parseBinaryGltf = require('./parseBinaryGltf');
+var addPipelineExtras = require('./addPipelineExtras');
+var loadGltfUris = require('./loadGltfUris');
+var Cesium = require('cesium');
+var defined = Cesium.defined;
+var DeveloperError = Cesium.DeveloperError;
+
+module.exports = readGltf;
+
+function readGltf(gltfPath, callback) {
+    var fileExtension = path.extname(gltfPath);
+    var filePath = path.dirname(gltfPath);
+
+    if (!defined(gltfPath)) {
+        throw new DeveloperError('Input path is undefined.');
+    }
```

Comment:
Put this check above since `fileExtension` and `filePath` expect it to be defined.